azext_iot/operations/hub.py

```diff
@@ -3301,3 +3305,173 @@ def _customize_device_tracing_output(device_id, desired, reported):
         ):
             output["isSynced"] = True
     return output
+
+# Topic Space Preview
+
+
+def iot_hub_topic_space_create(
+    cmd,
+    topic_name,
+    topic_template,
+    topic_type,
+    hub_name=None,
+    resource_group_name=None,
+    login=None,
+    auth_type_dataplane=None,
+):
+    discovery = IotHubDiscovery(cmd)
+    target = discovery.get_target(
+        hub_name=hub_name,
+        resource_group_name=resource_group_name,
+        login=login,
+        auth_type=auth_type_dataplane,
+    )
+    resolver = SdkResolver(target=target)
+    service_sdk = resolver.get_sdk(SdkType.service_sdk)
+
+    topic_type = TopicSpaceType(topic_type)
+    if topic_type == TopicSpaceType.HighFanout:
```

Comment:
I guess this is point in time. Weird to have a valid arg (HighFanout) that is not actually supported yet.

Reply:
It will be supported at some point